### DISCUSSION TITLE
Will the removal of dockershim from kubernetes break dind?

### DISCUSSION THREAD

**Discussion by stefangluszek** [actions/actions-runner-controller#1289] [Questions] [answered]
In kubernetes 1.24 dockershim component will be removed.
https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/
Will this break the dind that's used for with the github action runner pod?

**Comment by toast-gear** [comment, marked as answer]
https://github.com/actions-runner-controller/actions-runner-controller/discussions/1006
